### DISCUSSION TITLE
Google still display the link to my old github page

### DISCUSSION THREAD

**Discussion by acorrenson** [community/community#33608] [Other Feature Feedback, Questions, & Ideas]
Hi all 👋 ,
I recently changed my github username. Unfortunately, when I google my name, the first result still links to my old github page
and displays a 404 error... I updated all my social media so they now point to the new address, but the problem persists.
Is there anything I could do ?

Thaaanks ❤️

**Comment by DenverCoder1** [comment]
Google will eventually stop displaying pages if they are consistently showing 404 errors, so one option is to just wait for Google to remove it.

Another option could be to register the old username again with a new account and create a profile readme that tells people the new location of your profile. If you don't include your name on the profile, it won't show up as much in search results after the next time it is cached, but it won't go away completely.
